mito-ai/mito_ai/gemini_client.py

```diff
@@ -112,50 +113,46 @@ async def request_completions(
         response_format_info: Optional[ResponseFormatInfo] = None,
         message_type: MessageType = MessageType.CHAT
     ) -> str:
-        try:
-            # Extract system instructions and contents
-            system_instructions, contents = get_gemini_system_prompt_and_messages(messages)
-
-            # Get provider data for Gemini completion
-            provider_data = get_gemini_completion_function_params(
-                model=self.model if response_format_info else GEMINI_FAST_MODEL,
-                contents=contents,
+        # Extract system instructions and contents
```

Comment:
The only change here is that I removed the try block, again so we can handle the exceptions in the providers.py file